apps/server/src/modules/loadDb.ts

```diff
@@ -67,7 +67,7 @@ async function loadDb() {
   return { db, data };
 }
 
-export let db = {};
+export let db = {} as Low<DatabaseModel>;
```

Comment:
Does

```ts
export let db: Low<DatabaseModel> = {};
```

work?

Reply:
not really, we are cheating here and the code is less than ideal

we export a mutable variable (uncool) db. and then have an init sequence to create the lowdb object
this is to avoid race conditions on starting, we need to make sure the database gets initialised at a specific test

we could instead do 
`export let db = Low<DatabaseModel> | null = null`
but then we would need to pepper the whole code with nullchecks 

what do you think?

Reply:
Not worth it. Resolving.